packages/front-end/components/Features/SDKConnections/SDKLanguageSelector.tsx

```diff
@@ -16,44 +16,26 @@ const tabs: Record<LanguageFilter, string> = {
   edge: "Edge",
 };
 
-function LanguageOption({
+export function SDKLanguageOption({
   language,
   selected,
-  setValue,
-  multiple,
+  onClick,
 }: {
   language: SDKLanguage;
-  selected: Set<SDKLanguage>;
-  setValue: (languages: SDKLanguage[]) => void;
-  multiple: boolean;
+  selected: boolean;
+  onClick: (language: React.MouseEvent) => void;
```

Comment:
I would make this accept no arguments and just always do `e.preventDefault()` internally before calling the provided `onClick()` method.  I can't think of a reason why a consumer of this component would need access to the raw mouse event.